tests/integration/service_test.py

```diff
@@ -699,6 +699,13 @@ def test_empty_labels(self):
         for name in labels_list:
             self.assertIn((name, ''), labels)
 
+    def test_custom_container_name(self):
```

Comment:
Discussed this in person. For the record, we can look at increasing tests after code freeze depending on priority of other tasks.